core/templates/pages/admin-page/roles-tab/admin-roles-tab.component.ts

```diff
@@ -71,7 +76,9 @@ export class AdminRolesTabComponent implements OnInit {
   }
 
   removeRole(roleToRemove: string): void {
-    this.roleInUpdate = roleToRemove;
+    // Updating roleCurrentlyBeingUpdatedInBackend as the given roleToRemove
```

Comment:
I didn't mean to add the comment here. Add it above line 158 to explain what the property represents, then you can delete this one and the one on line 120.

Reply:
Done!